<ui version="4.0" >
 <class>TabPatch</class>
 <widget class="QWidget" name="TabPatch" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Patch</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>2</number>
       </property>
       <property name="topMargin" >
        <number>2</number>
       </property>
       <property name="rightMargin" >
        <number>2</number>
       </property>
       <property name="bottomMargin" >
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel" >
         <property name="text" >
          <string>Diff to:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonParent" >
         <property name="focusPolicy" >
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Check to diff against parent</string>
         </property>
         <property name="text" >
          <string>P&amp;arent</string>
         </property>
         <property name="shortcut" >
          <string>Alt+A</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonHead" >
         <property name="focusPolicy" >
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Check to diff against HEAD</string>
         </property>
         <property name="text" >
          <string>H&amp;EAD</string>
         </property>
         <property name="shortcut" >
          <string>Alt+E</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonSha" >
         <property name="focusPolicy" >
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="toolTip" >
          <string>Use CTRL + right click to select a revision to diff against</string>
         </property>
         <property name="text" >
          <string>&amp;SHA</string>
         </property>
         <property name="shortcut" >
          <string>Alt+S</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditDiff" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>SHA value of selected revision. Use CTRL + right click to select.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonFilterPatch" >
         <property name="toolTip" >
          <string>Toggle view of added and removed lines (CTRL+H)</string>
         </property>
         <property name="shortcut" >
          <string>Ctrl+H</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="RevDesc" name="textBrowserDesc" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
        <widget class="PatchContent" name="textEditDiff" >
         <property name="undoRedoEnabled" >
          <bool>false</bool>
         </property>
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags" >
          <set>Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </widget>
       <widget class="FileList" name="fileList" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>FileList</class>
   <extends>QListWidget</extends>
   <header>filelist.h</header>
  </customwidget>
  <customwidget>
   <class>RevDesc</class>
   <extends>QTextBrowser</extends>
   <header>revdesc.h</header>
  </customwidget>
  <customwidget>
   <class>PatchContent</class>
   <extends>QTextEdit</extends>
   <header>patchcontent.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >revdesc.h</include>
 </includes>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections/>
</ui>
